src/devtools/client/webconsole/actions/filters.js

```diff
@@ -14,41 +14,45 @@ const {
   FILTERS,
 } = require("devtools/client/webconsole/constants");
 
-function filterTextSet(text) {
-  return {
-    type: FILTER_TEXT_SET,
-    text,
+export function filterTextSet(text) {
+  return ({ dispatch, getState }) => {
+    const filtersState = { ...getAllFilters(getState()), text }
+    return dispatch({
+      type: FILTER_TEXT_SET,
+      filtersState,
```

Comment:
Why add the filtersState to the payload? It doesn't look like it's being handled by the reducer (`reducers/filters.js`)